### PR TITLE
Provides a fix for ticket #29

### PR DESCRIPTION
**Pull request by tribeiro**
The telescope would lost connection after 2 minutes of inactivity. Also solves some minor issues. Specially from time to time there would occur some error with the connection to the telescope while slewing to an object causing chimera-tel, for instance, to crash. But, if you persist with the connection it would come back.

As requested, I checked out the latest version of chimera from astroufsc and uploaded the changes.

### CODE REVIEW

**Comment by ankanaan** [comment]
howdy,

so, this

"But, if you persist with the connection it would come back."

means that somehow you keep the connection open.  By doing what?  You ask
something to the telescope every minute or so?  Just curious.

Antonio

**Comment by tribeiro** [comment]
When you move the telescope to and object chimera-tel enters in a loop until the telescope is positioned. During this loop it keeps asking the position of the telescope to astelos. I guess that, from time to time it gets a corrupted packaged which would cause chimera-tel to raise an exception and close. But nothing special actually happened. So, now it stores the coordinates each loop and, when it gets the corrupted packed it just ignore it. If something actually bad happens the track will timeout eventually.

**Comment by nsaavedra-naircl** [comment]
Correct me if I'm wrong: the lingo used here is TPL2? IF so, it is clear
text optionally and we should be able to snoop on it. IMHO, an issue like
this should be debugged...and no, I am not volunteering!

On Sat, Jan 24, 2015 at 10:52 AM, Tiago <notifications@github.com> wrote:

> When you move the telescope to and object chimera-tel enters in a loop
> until the telescope is positioned. During this loop it keeps asking the
> position of the telescope to astelos. I guess that, from time to time it
> gets a corrupted packaged which would cause chimera-tel to raise an
> exception and close. But nothing special actually happened. So, now it
> stores the coordinates each loop and, when it gets the corrupted packed it
> just ignore it. If something actually bad happens the track will timeout
> eventually.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/astroufsc/chimera/pull/31#issuecomment-71317560>.
>



-- 
Nelson Saavedra Pinos
------------------------------------------------
NAIR Servicios Informáticos E.I.R.L.
------------------------------------------------
e: saavedra.nelson@gmail.com
c: +56 51 69074027

**Comment by tribeiro** [comment]
Agreed... I'll take a more careful look at it later... 

Sent from mobile

> Em 24/01/2015, às 12:10, Nelson Saavedra <notifications@github.com> escreveu:
> 
> Correct me if I'm wrong: the lingo used here is TPL2? IF so, it is clear 
> text optionally and we should be able to snoop on it. IMHO, an issue like 
> this should be debugged...and no, I am not volunteering! 
>

File: src/chimera/instruments/astelco.py [hunk 1]
```diff
@@ -94,8 +103,8 @@ def __init__(self):
                 self._calibration[rate][direction] = 1
 
         self._user = "admin"
-        self._password = "admin"
-        self._aahost = "localhost"
+        self._password = "admin" 
+        self._aahost = "192.168.10.3"
```
Comment (wschoenell) [inline src/chimera/instruments/astelco.py]: This part should be on the config file, no?

Comment (ankanaan) [inline src/chimera/instruments/astelco.py]: yes, it should indeed.

On Wed, Jan 28, 2015 at 3:55 PM, William Schoenell <notifications@github.com

**Comment by tribeiro** [comment]
Does anyone knows how to do it? I tried as much as I could...

**Comment by ankanaan** [comment]
by imitation :-)

you could check telescope.py and meade.py which for sure do that.  If
nobody here knows it is time to ask PH and document the process.

A

**Comment by nsaavedra-naircl** [comment]
You mean puth that stuff into config? I can do it (or try at least), once
I'm off this teething FLI thing.

On Fri, Jan 30, 2015 at 7:13 AM, Antonio Kanaan <notifications@github.com>
wrote:

> by imitation :-)
>
> you could check telescope.py and meade.py which for sure do that. If
> nobody here knows it is time to ask PH and document the process.
>
> A
>
> On Fri, Jan 30, 2015 at 7:13 AM, Tiago <notifications@github.com> wrote:
>
> > Does anyone knows how to do it? I tried as much as I could...
> >
> > —
> > Reply to this email directly or view it on GitHub
> > <https://github.com/astroufsc/chimera/pull/31#issuecomment-72173559>.
> >
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/astroufsc/chimera/pull/31#issuecomment-72180049>.
>



-- 
Nelson Saavedra Pinos
------------------------------------------------
NAIR Servicios Informáticos E.I.R.L.
------------------------------------------------
e: saavedra.nelson@gmail.com
c: +56 51 69074027

**Comment by ankanaan** [comment]
by the way, the HOWTO for this was written ages ago by one such Kanaan:

https://code.google.com/p/chimera/wiki/Chimera_config

**Comment by tribeiro** [comment]
Of course it was something stupid! I was trying to access the information on __init__ while it is only available after __star__.... Will update it in a minute...

**Pull request merged**